hms-app/src/components/input-forms/EditUserForm.tsx

```diff
@@ -32,28 +31,20 @@ export default function EditUserForm(props: IProps) {
   })
 
   const loadSelectedUser = () => {
-    getUserDetails(instance, userId).then(
-      (data) => {
-        setIsLoading(false)
-        setUser({
-          id: userId,
-          lastName: data.lastName,
-          firstName: data.firstName,
-          emailAddress: data.emailAddress,
-          roles: data.roles,
-          imageUrl: data.imageUrl,
-        })
-      },
-      () => {
-        setIsLoading(false)
-      }
-    )
+    setIsLoading(false)
+    setUserData({
+      id: user.id,
+      lastName: user.lastName as any,
+      firstName: user.firstName,
+      emailAddress: user.emailAddress,
+      roles: user.roles as any,
+      imageUrl: user.imageUrl as any,
+    })
```

Comment:
Does it make sense to set the `userData` after getting the `user` from the `UserDetails`/`HeaderMenu` ?

Because to edit a user, we already have `editUser` function in `userAction`. And this function only works with the specific properties like in the userData that we set

Reply:
yes we dont need the getUserDetails function anymore when we pass down the whole user